pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 description: Automatic sourcecode generation for Dart
 homepage: https://github.com/dart-lang/source_gen
 environment:
-  sdk: '>=1.12.0 <2.0.0'
+  sdk: '>=1.23.0-dev.11.0 <2.0.0'
```

Comment:
I think we should bump the min sdk version but not do a major version bump in this package (unless there is a breaking change I am missing).

There isn't much of a point in generating code that can't run in your current sdk imo, it would just lead to confusion.